File: URI_Online_Judge_1079.c
#include <stdio.h>
 
int main() {
 
    int N,i;
    float a,b,c,avg;
    
    scanf("%d",&N);
    for(i=0; i<N; i++){
    	scanf("%f", &a);
  	scanf("%f", &b);
  	scanf("%f", &c);
    	avg = (a*2 + b*3 + c*5)/10;
    	printf("%.1f\n", avg);
	}
    
 
    return 0;
}

File: URI_Online_Judge_1079.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {

		int N;
		double a, b, c, avg;

		Scanner in = new Scanner(System.in);

		N = in.nextInt();
		for (int i = 0; i < N; i++) {
			a = in.nextDouble();
			b = in.nextDouble();
			c = in.nextDouble();

			avg = (a * 2 + b * 3 + c * 5) / 10;

			System.out.printf("%.1f\n", avg);

		}
	}

}
